<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#formitem{
	border-radius:18px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,3,2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>10</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color:blue;
font-size: 25pt;
font-weight:bold;</string>
        </property>
        <property name="text">
         <string>FeiYoung Connector</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="tips">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>10</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 0, 0);
font-weight:bold;
font-size:15pt;</string>
        </property>
        <property name="text">
         <string>仅个人研究学习使用 不得用于非法用途</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	height:50%;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,1,2">
      <item>
       <widget class="QLineEdit" name="phoneNumber">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>20</horstretch>
          <verstretch>30</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:38px;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>输入手机账号</string>
        </property>
        <property name="cursorMoveStyle">
         <enum>Qt::LogicalMoveStyle</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mac">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>20</horstretch>
          <verstretch>30</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:38px;
</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Mac地址 会自动获取</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ip">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>20</horstretch>
          <verstretch>30</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:38px;
</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>分配IP 会自动获取</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nasIp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>20</horstretch>
          <verstretch>30</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:38px;
</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>nasIP 自动获取不到就手工填写</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="customContent">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>20</horstretch>
          <verstretch>45</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:38px;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>自定义登录的时候使用 其他时候不用</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border:1px solid;
	border-radius:50px;

	height:50%;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1">
      <item>
       <widget class="QPushButton" name="getInfo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>15</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #fff;
background-color: #f0ad4e;
border-color: #eea236;</string>
        </property>
        <property name="text">
         <string>获取信息</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="auth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>10</horstretch>
          <verstretch>15</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #fff;
    background-color: #5cb85c;
    border-color: #4cae4c;</string>
        </property>
        <property name="text">
         <string>开始认证</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="costomAuth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>15</horstretch>
          <verstretch>15</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #fff;
    background-color: #d9534f;
    border-color: #d43f3a;</string>
        </property>
        <property name="text">
         <string>自定义认证</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
